### ISSUE TITLE
Item Collector does not work with Functional Storage Controller Access Point

### ISSUE THREAD

**Issue by Rafiki2085** [Direwolf20-MC/JustDireThings#215]
### Just Dire Things version

1.4.2

### Minecraft Version

1.21.1

### (Neo)Forge Version

21.1.61

### Modpack & Version

Direwolf20 1.21 version 1.2.1

### Do you have optifine or Rubidium installed?

None

### Describe the issue

Placing a Item Collector on a Functional Storage Controller Access Point does not it to collect items

### Steps to reproduce

1. Create & Place Storage Drawer, Controller, and Controller Access Point
2. Binding them all together
3. Lock Contents of Drawers and set to an item
4. Place Item Collector on Access Point
5. Drop Item In Range of Access Point
6. Item will not be picked up

### Expected behaviour

Item should be picked up and placed in the drawer same as when the Item Collector is on a Controller

**Comment by delta2k** [comment]
i don't think it's a problem with the collector. It works for me. The problem is with the access points have to be placed after you placed the controller and draw and link them. otherwise it shows no items in the drawer system. this happened to me.  you could try that and report back.

**Comment by Rafiki2085** [comment]
Yes, I have a controller placed first and the access point linked before placing the item collector, I can also see the thebfluid content that is a part of my drawer system on the access point overlay.
